cmd/goal/clerk.go

```diff
@@ -146,6 +147,7 @@ func init() {
 
 	simulateCmd.Flags().StringVarP(&txFilename, "txfile", "t", "", "Transaction or transaction-group to test")
 	simulateCmd.Flags().StringVarP(&outFilename, "outfile", "o", "", "Filename for writing simulation result")
+	simulateCmd.Flags().BoolVar(&unlimitLog, "unlimit-log", false, "Remove limit on log opcode usage in simulation")
```

Comment:
Since unlimited logs is not achievable, maybe "raise-log-limits" or "high-log-limits" might be a better name?

Reply:
(same applying to internal code name on this option?)

Reply:
I like it

Reply:
see [unlimitLog to liftLogLimits](https://github.com/algorand/go-algorand/pull/5247/commits/af282df7ff135b32f83f4c99e81dc3d13eb97694)